core/src/sql/statements/define/user.rs

```diff
@@ -25,13 +25,15 @@ pub struct DefineUserStatement {
 	pub hash: String,
 	pub code: String,
 	pub roles: Vec<Ident>,
+	#[revision(start = 2)]
```

Comment:
Shouldn't we bump the `revision` to `3` and then make this `start = 3`?

Reply:
Oh! I assumed that all changes that would be released under `2.0` would be included in the same revision, as `2.0` has not yet been released. Should we count unreleased changes in revisions? For example, if we change the structure after this PR is merged, should we add a new revision?

In any case, I see now that `if_not_exists` is already on `1.x` so a new revision is at least required for this change. I will make a commit to update it. Thanks a lot Rushmore!

Reply:
> I assumed that all changes that would be released under 2.0 would be included in the same revision, as 2.0 has not yet been released.

Yes, technically that's fine. But as you found out, trying to do that is error prone. If you forget or overlook to check the current version of the released struct, you risk adding the field to a version which is already released. I think it's safer to just increment the version whenever adding a new field.

Reply:
Thank you @rushmorem! Fixed in [cde0200](https://github.com/surrealdb/surrealdb/pull/4116/commits/cde0200d8ee63b294a0415b3be4edd57ec15e2a0).